csr/csr.go

```diff
@@ -50,6 +50,12 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 	if err := csr.CheckSignature(); err != nil {
 		return errors.New("invalid signature on CSR")
 	}
+	if len(csr.EmailAddresses) > 0 {
+		return errors.New("CSR contains one or more email address fields")
```

Comment:
Since the nit was small and sounded sensible, and @jsha is OOO this week, I resolved this on his behalf with d81e51d.